client/analytics/report/orders/chart.js

```diff
@@ -6,16 +6,19 @@ import { __ } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
 import { find } from 'lodash';
 import PropTypes from 'prop-types';
+import { applyFilters } from '@wordpress/hooks';
 
 /**
  * Internal dependencies
  */
 import ReportChart from 'analytics/components/report-chart';
 import ReportSummary from 'analytics/components/report-summary';
 
+const CHARTS_FILTER = 'woocommerce-charts-list';
```

Comment:
We should use `_` instead of `-` for hooks, to follow the same as WooCommerce core.

Reply:
Since these are JS hooks, not PHP hooks, I don't think the naming standards are the same. AFAIK there isn't anything written yet, but [the examples](https://wordpress.org/gutenberg/handbook/extensibility/extending-blocks/) [tend to use](https://wordpress.org/gutenberg/handbook/extensibility/extending-editor/) `-` or camelCase.

Probably this should actually be `woocommerce.getCharts`, looking at some of the existing gutenberg hooks.